alfresco-docker-transform-misc/src/main/java/org/alfresco/transformer/transformers/AppleIWorksContentTransformer.java

```diff
@@ -76,15 +76,6 @@
     //                (225 x 173) preview-web.jpg
     //                 (53 x  41) preview-micro.jpg
 
-    @Override
-    public boolean isTransformable(String sourceMimetype, String targetMimetype,
-        Map<String, String> parameters)
-    {
-        return MIMETYPE_IWORK_KEYNOTE.equals(sourceMimetype) ||

```

Comment:
I can't comment on the actual import statement above, but there are unused imports for the MIMETYPE_* fields left behind in the transformers (this one and others).